### PR TITLE
Notification ralph dev

### PR DESCRIPTION
**Pull request by zli213**

### CODE REVIEW

**Comment by ryanwooa** [comment]
- Color please match our theme color 
- There is no notification for completing the task
- After creating a new task, click on the project link in the prompt and the corresponding page cannot be found

**Comment by ryanwooa** [comment]
image
Prompt once for every modification of a date?

**Comment by ryanwooa** [comment]
- Why not close notification after a period of time?
- There is no notification for completing the task
- About width of notification, if its better to let it not beyond left bar when its content is not too long?
image

File: components/pages/AppPages/Inbox.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
 "use client";
 
 import TodoList from "../../application/widgets/TodoList";
-import { useEffect } from "react";
+import { useState, useEffect, use } from "react";
```
Comment (ryanwooa) [inline components/pages/AppPages/Inbox.js:4]: "use"?

File: components/pages/AppPages/Labels.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
 "use client";
 
 import styles from "../../../styles/scss/application.module.scss";
-import { useEffect, useState } from "react";
-import Icon from "../../application/widgets/Icon";
+import { use, useEffect } from "react";
```
Comment (ryanwooa) [inline components/pages/AppPages/Labels.js:4]: "use"

**Comment by ryanwooa** [comment]
image
- How about limit the width of notification.
- How about not show the detail of task when complete the task
- There only can show one notification anytime. Should we consider displaying multiple notifications when there are consecutive tasks completed. For example, when a new notification appears below, the old one moves upwards

**Comment by ryanwooa** [comment]
Should we consider making the notification text smaller

**Comment by zli213** [comment]
> image
> 
> * How about limit the width of notification.
> * How about not show the detail of task when complete the task
> * There only can show one notification anytime. Should we consider displaying multiple notifications when there are consecutive tasks completed. For example, when a new notification appears below, the old one moves upwards

I believe displaying a certain number of characters helps differentiate between notifications, otherwise a bunch of identical notifications seem a bit redundant. Therefore, based on your suggestion, I've limited the number of characters that can be displayed, and the effect is as follows. Similarly, I adjusted the font size, you can check if it's suitable.
As for displaying multiple notifications at once, actually, that's the simplest approach, but currently, it's specifically designed to only show one notification at a time to achieve the same display effect as the `todoist page`. If everyone prefers that effect, we can consider changing it back in the future.
CleanShot 2024-02-09 at 16 32 37@2x
CleanShot 2024-02-09 at 16 32 48@2x
CleanShot 2024-02-09 at 16 32 55@2x

File: components/pages/AppPages/Inbox.js (hunk 1 above)
Comment (zli213) [reply components/pages/AppPages/Inbox.js]: Delete

File: components/pages/AppPages/Labels.js (hunk 2 above)
Comment (zli213) [reply components/pages/AppPages/Labels.js]: Delete

**Review by ryanwooa — APPROVED** [review]

**Pull request merged**